depthai_sdk/src/depthai_sdk/nn_models/facial_landmarks_68_160x160/config.json

```
@@ -0,0 +1,11 @@
+{
+    "model": {
+        "model_name":"facial_landmarks_68_160x160",
+        "zoo": "depthai"
+    },
+    "nn_config": {
+        "output_format" : "raw"
+    },
+    "handler": "handler.py",
```

2022.1 doesn't work afaik. todo: test
```suggestion
    "handler": "handler.py",
    "openvino_version": "2021.4",
```